bundle-core/src/main/java/net/discdd/client/bundletransmission/BundleTransmission.java

```diff
@@ -197,10 +188,6 @@ private BundleDTO generateNewBundle(String bundleId) throws RoutingExceptions.Cl
 
         } catch (InvalidMessageException e) {
             throw new IOException("Error processing message: " + e.getMessage(), e);
-        } finally {
```

Comment:
we want to keep this to make sure the subtask gets cleaned up if an error happens.